controller/token.go

```diff
@@ -290,7 +306,12 @@ func (c *TokenController) Delete(ctx *app.DeleteTokenContext) error {
 			return err
 		}
 		if len(tokens) > 0 {
-			return appl.ExternalTokens().Delete(ctx, tokens[0].ID)
+			for _, token := range tokens {
```

Comment:
This situation shouldn't arise in a ideal world , right?

Reply:
That’s right. But theoretically it may happen if there are many concurrent link requests from the same users (or even GET token requests until we stop get token from KC first and then save them in DB).